packages/validator/src/processor.ts

```diff
@@ -43,7 +43,7 @@ export function processSpecs(
         }
       }
 
-      break;
```

Comment:
Let me try and understand this fix: is this for when you have multiple `__optional` keys in the format file? If so: is it legal to have duplicate keys in a json file? i.e. it's valid?

Reply:
`__ignore` is a bit hidden keyword, that validator is using internally when working with nested templates. I guess I accidentally submitted it in this commit, it should have been separate bug fix commit.
This section will ensure, that validator won't print any extra parameter messages, we want this because that part of specification should be already validated by nested template. However if we `break` and don't check any further parameters in template, then validation wouldn't be complete. Which means there must be a `continue` so that everything is validated.
Templates are normal json, which does not allow duplicated keywords, but this change is not about that.